### ISSUE TITLE
Increase color contrast of some texts and links

### ISSUE THREAD

**Issue by sharifsalah** [google/docsy#41]
Some text and links in the default Docsy styles result in low contrast. [According to WCAG](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=143#qr-visual-audio-contrast-contrast), text should have a contrast ratio of at least 4.5:1. 

The most noticeable example of poor contrast I've noticed so far is on the [/community page in the example site](https://testydocsy.netlify.com/community/). The contrast of the links in the middle section against the background is 3.17:1.

'Out of the box', it would be great if the Docsy example and theme demonstrated a design that maintained a contrast ratio of 4.5:1 throughout. There are various tools that can automate checking contrast across a site or page.
